tests/ops.rs

```diff
@@ -265,3 +265,31 @@ fn matrix_not() {
     assert_eq!(!a, matrix![-2, 2; -4, 6]);
     assert_eq!(!&a, matrix![-2, 2; -4, 6]);
 }
+
+mod matrix_multiplication {
+    use super::*;
```

Comment:
For consistency, I would prefer no module and prefix each of these test functions with `matrix_mul_`